### DISCUSSION TITLE
Dual boot: use same the Windows folder?

### DISCUSSION THREAD

**Discussion by ezhan** [abraunegg/onedrive#2858] [Q&A] [answered]
I want to install Linux beside Windows and I'm curious, is it possible to use for Onedrive on Linux same Onedrive folder on the Windows partition?

**Comment by abraunegg** [comment, marked as answer]
@ezhan 
>  is it possible to use for Onedrive on Linux same Onedrive folder on the Windows partition?

There are too many complications with that:

1. Windows 'on-demand' functionality that is problematic for Linux
2. NTFS / BitLocker
3. Need to use other utilities to 'read|write' the Windows partition

You have a number of options:
1. Dont use 'on-demand' and sync your data to a separate drive / directory formatted with FAT32 which will then work with this client
2. Use https://github.com/gbrielgustavo/ntfs-3g-onedrive/ to read|write your Windows partition. Please read https://github.com/abraunegg/onedrive/discussions/2531 for details - but depends on you also most likely disabling 'on-demand' for Windows
3. Use 'on-demand' and https://github.com/jstaf/onedriver

**Reply by ezhan** [reply]
Thank you very much for answering!

Why is the NTFS a problem? I have setup Thunderbird to use same profile on the NTFS partition, that TB under Windows uses and it seems to work normally.

**Reply by abraunegg** [reply]
Please read the above links as to *why*

**Reply by ezhan** [reply]
Sorry, my bad. :( Now all is clear.
